rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -391,6 +391,14 @@ public void registerBtcTransaction(
                 throw new RegisterBtcTransactionException("Transaction already processed");
             }
 
+            if (svpIsOngoing() && isTheSvpSpendTransaction(btcTx)) {
+                markTxAsProcessed(btcTx);
+                saveNewUTXOs(btcTx);
+                provider.setSvpSpendTxHashUnsigned(null);
+                // proceed with svp success
```

Comment:
Why not put this into a method, as the task description said? The task description and PR title give the impression that a new method will be added to do this.

Reply:
I changed the task description since it was not accurate, thanks!

Reply:
updated